src/Tizen.NUI/src/public/WebView/WebBackForwardList.cs

```diff
@@ -188,7 +188,7 @@ public WebBackForwardListItem GetItemAtIndex(int index)
         /// Get copied backward items.
         /// </summary>
         /// <param name="index">limit The number of items to retrieve.</param>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 9 </since_tizen>
         public WebBackForwardSubList GetBackwardItems(int index)
```

Comment:
If possible, I think we could return `IList<WebBackForwardListItem>` and remove class `WebBackForwardSubList`.

Reply:
Done.